index.html

```diff
@@ -84,6 +84,148 @@ <h5 class="text-purple skillboxPL">Skills</h5>
                     <li><a href="#"><i class="fab fa-medium"></i></a></li>
                 </ul>
             </section>
+            <section class="section-work">
+                <div class="generic-grid-container">
+                    <div class="headWorkTitlecont">
+                        <i class="fas fa-arrow-down"></i>
+                        <div>Work</div>
+                    </div>
+                </div>
+                <div class="work-container">
+                    <div class="workCard-container">
+                        <div class="header-work">MY RECENT WORK</div>
+                        <hr class="workseparator">
+                    </div>
+                    <div class="workCard-container">
+                        <img class="image-work elipse19" src="src/img/img_Placeholder.png" alt="Image Placeholder">
+                        <div class="title-work">Multi-Post Stories</div>
+                        <div class="content-work">
+                            A daily selection of privately 
+                            personalized reads; no accounts 
+                            or sign-ups required. has been the 
+                            industry's standard dummy text 
+                            ever since the 1500s, when an 
+                            unknown printer took a standard 
+                            dummy text.
+                        </div>
+                        <div class="langFrame-work">
+                            <div class="lang-box separator-lang" id="css">
+                                <h4 class="language-text">css</h4>
+                            </div>
+                            <div class="lang-box separator-lang" id="html">
+                                <h4 class="language-text">html</h4>
+                            </div>
+                            <div class="lang-box separator-lang" id="bootstrap">
+                                <h4 class="language-text">bootstrap</h4>
+                            </div>
+                            <div class="lang-box" id="ruby">
+                                <h4 class="language-text">Ruby</h4>
+                            </div>
+                        </div>
```

Comment:
- [ ] **[REQUIRED CHANGES]** When we're dealing with similar or identical items that will be displayed together, we should try to wrap them into a `<ul>` or `<ol>` tag, to make them a list. Could you change it for us?